### DISCUSSION TITLE
Cannot save profiles :(

### DISCUSSION THREAD

**Discussion by Perfectionman** [opentrack/opentrack#1404] [Q&A]
Running OpenTrack 2022.1.0     After closing OpenTrack I lose all settings and when I relaunch it everything is back to default settings, NOTHING saved.  I have done uninstalls and made sure to delete all files and reinstalled.  I have tried the portable version and same result.  I guess the last option is to try an older version and see if same results.

**Comment by sthalik** [comment]
Disable OneDrive sync for the Documents directory.

**Reply by Perfectionman** [reply]
I unchecked OneDrive sync for the Documents directory, restarted the pc but no joy :(
